### ISSUE TITLE
Restoring app loses current place

### ISSUE THREAD

**Issue by dracan** [slapperwan/gh4a#352]
If you background the app, then bring it back to the foreground, it'll lose your current place and take you back to the start page.

**Comment by maniac103** [comment]
Couple of questions:
- What Octodroid version are you using?
- Where were you in the app when you backgrounded it?
- During being backgrounded, is the app killed by Android and restarted? You can tell by checking whether the black window with 'Octodroid' title appears shortly on foregrounding - if yes, it was killed before.

**Comment by dracan** [comment]
Hi.

* I'm using version 4.0.3
* It was when I was navigating a repository - I'd seen it happen both when viewing a file and on the directory listing page. I'd also seen it when replying to this issue - I backgrounded, then had lost what I had written. Strangely though, I've just tried explicitly killing the app, and this seems to have fixed the issue. It's now restoring correctly on each of the above mentioned screens. Very odd.
* I can't test this now as I can no longer reproduce this (see my comments in the last bullet point). I can't see why it would have been killing the process when backgrounding it though.

As this seems to have fixed itself after killing it, I guess this ticket's closable?

Whilst I'm on, I just want to say what a great app! I wanted to find an alternative for the official Github one, which is absolutely appalling! Very impressed with OctoDroid - good work! :)
